pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2782,13 +2782,33 @@ cdef class Week(SingleConstructorOffset):
     Parameters
     ----------
     weekday : int or None, default None
-        Always generate specific day of week. 0 for Monday.
+        Always generate specific day of week.
+        0 for Monday and 6 for Sunday.
 
-    Examples
+    See Also
     --------
-    >>> ts = pd.Timestamp(2022, 1, 1)
-    >>> ts + pd.offsets.Week()
-    Timestamp('2022-01-08 00:00:00')
+    pd.tseries.offsets.WeekOfMonth :
+     Describes monthly dates like, the Tuesday of the
+     2nd week of each month.
+
+    Examples
+    ---------
```

Comment:
just noticed, I think the underscores need to be the same length as the title

could you please also build this docstring locally and screenshot it and check it looks alright?

Reply:
<img width="627" alt="image" src="https://user-images.githubusercontent.com/19975437/212780302-9f9b11d7-c1e0-4904-80e6-f75b9840ce75.png">
Hi @MarcoGorelli, it looks like this. It looks fine to me :D but should i add one more commit?

Reply:
that's not the rendered docstring, see https://pandas.pydata.org/docs/dev/development/contributing_documentation.html#building-the-documentation for how to build it

something like
```console
cd doc
python make.py clean && python make.py --single pandas.tseries.offsets.Weekday
```

Reply:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/19975437/212901101-32a3cae9-21dd-4361-86c1-ddced19cc4a4.png">
Hey @MarcoGorelli , this is the rendered docstring, does this work?